packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.styles.ts

```diff
@@ -3,6 +3,7 @@ import { getGlobalClassNames, FontSizes, AnimationVariables } from '../../Stylin
 
 const GlobalClassNames = {
   root: 'ms-GroupedList',
+  compact: 'ms-GroupedList--Compact',
```

Comment:
super nit, but for consistency within these class names I'd prefer `--compact`

Reply:
Fixed!